modules/gapi/test/cpu/gapi_core_tests_fluid.cpp

```diff
@@ -19,8 +19,6 @@ namespace opencv_test
 INSTANTIATE_TEST_CASE_P(MathOpTestFluid, MathOpTest,
                         Combine(Values(CV_8UC3, CV_8UC1, CV_16SC1, CV_32FC1),
                                 Values(cv::Size(1920, 1080),
-                                       cv::Size(1280, 720),
-                                       cv::Size(640, 480),
```

Comment:
This selection looks strange. Shouldn't the `1080` option be removed & `720` be kept as in the previous file?
As far I can see, the below cases also instantiate `720` and not `1080`.

Reply:
Done